src/components/Articles.jsx

```diff
@@ -1,28 +1,33 @@
-import { useState, useEffect} from "react"
-import getArticles from "../../api-get"
+import { useState } from "react"
+import {useEffect} from "react"
+import {getArticles} from "../../api-get"
+import '../../src/Styling/article.css'
+
 
 export const Articles = () => {
     
-    const [articleList, setArticleList] = useState([])
-
-    useEffect (() => {
-        getArticles()
-        .then((body) =>{
-            console.log(body)
-            setArticleList(body)
-            console.log(body)
-        })
-        .catch((err) =>{
-            console.error(err)
-        })
-    }, [])
+const [articleList, setArticleList] = useState([])
 
-    return (
-        <div>
-            {articleList.map((article) =>{
-                <p>{article}</p>
-                console.log('article', article)
-            })}
-        </div>
-    )
+useEffect (() => {
+    getArticles()
+    .then((body) =>{
+        setArticleList(body.articles)
+    }).catch((err) =>{
+        console.error(err)
+    })
+}, [])
+return (
+    <div className="articles-container">
+        {articleList.map((article) => (
+            <div className="article-card" key={article.article_id}>
+                <img className="article-image" src={article.article_img_url} alt={article.title} />
+                <h2 className="article-title">{article.title}</h2>
```

Comment:
We also want to see when the articles were created in a human readable format as they should already be sorted by date when they come back from the api, as well as the comment count - see this on how to display this info: https://www.refactoringui.com/previews/labels-are-a-last-resort